0.6.0 CTS Docs (#12913)
hashicorp/consul: website/content/docs/nia/cli/index.mdx

## Daemon

> ~> Running CTS as a daemon without using a command is deprecated in CTS 0.6.0 and will be removed at a much later date in a major release. For information on the preferred way to run CTS as a daemon see the [`start` command docs](/docs/nia/cli/start)

Author: ```suggestion
~> Running CTS as a daemon without using a command is deprecated in CTS 0.6.0 and will be removed at a much later date in a major release. For information on the preferred way to run CTS as a daemon review the [`start` command docs](/docs/nia/cli/start)
```

> When running as a daemon, there is no default configuration to run CTS in a meaningful way. Setting a configuration flag `-config-file` or `-config-dir` is required. For example:

Author: When CTS runs as a daemon, there is no default configuration to start CTS. You must set a configuration flag `-config-file` or `-config-dir`. For example:

> When running as a daemon, there is no default configuration to run CTS in a meaningful way. Setting a configuration flag `-config-file` or `-config-dir` is required. For example:

Author: ```suggestion
When CTS runs as a daemon, there is no default configuration to start CTS. You must set a configuration flag -config-file or -config-dir. For example:
```

> To view a list of available flags, use the `-help` or `-h` flag.

Author: ```suggestion
To review a list of available flags, use the `-help` or `-h` flag.
```

## Commands

> In addition to running the daemon, CTS has a set of commands that act as a client to the daemon server. The commands provide a user-friendly experience interacting with the daemon. The commands use the CTS APIs but does not correspond one-to-one with it. Please see the individual commands in the left navigation for more details.

Author: ```suggestion
In addition to running the daemon, CTS has a set of commands that act as a client to the daemon server. The commands provide a user-friendly experience interacting with the daemon. The commands use the CTS APIs but does not correspond one-to-one with it. Please review the individual commands in the left navigation for more details.
```

consul-terraform-sync task disable -http-addr=http://localhost:2000 task_a
```

## Autocompletion

The `consul-terraform-sync` command features opt-in autocompletion for flags, subcommands, and
arguments (where supported).

To enable autocompletion, run:
```shell-session

> $ consul-terraform-sync -autocomplete-install

Author: ```suggestion
$ consul-terraform-sync -autocomplete-install
```
```

consul-terraform-sync task disable -http-addr=http://localhost:2000 task_a
```

## Autocompletion

The `consul-terraform-sync` command features opt-in autocompletion for flags, subcommands, and
arguments (where supported).

> Enable autocompletion by running:
> 
> ```shell-session
> $ consul-terraform-sync -autocomplete-install
> ```

Author: ```suggestion
To enable autocompletion, run:
```shell-session
$ consul-terraform-sync -autocomplete-install
```

> Enable autocompletion by running:
> 
> ```shell-session
> $ consul-terraform-sync -autocomplete-install
> ```
> 
> ~> Be sure to **restart your shell** after installing autocompletion!

Author: ```suggestion
To enable autocompletion, run:

```shell-session
$ consul-terraform-sync -autocomplete-install
```

~> After you install autocomplete, you must restart your shell for the change to take effect.
```

> ~> Be sure to **restart your shell** after installing autocompletion!

Author: ```suggestion
~> After you install autocomplete, you must restart your shell for the change to take effect.
```

> When you start typing a CTS command, press the `<tab>` character to show a

Author: ```suggestion
When you start typing a CTS command, press the `<tab>` key to show a
```

> list of available completions. Type `-<tab>` to show available flag completions.
> 
> If the `CTS_*` environment variables are set, the autocompletion will
> automatically query the running CTS server and return helpful argument suggestions.
> 

Author: ```suggestion
list of available completions. To show available flag completes, type `-<tab>`.

If the `CTS_*` environment variables are set, the autocompletion
automatically queries the running CTS server and returns helpful argument suggestions.

```
